.github/workflows/actions/remove-extra-firewall-rules/action.yml

```diff
@@ -0,0 +1,36 @@
+name: Remove extra firewall rules
+
+inputs:
+  AzureCredentials:
+    description: Azure Credentials
+    required: true
+  ResourceGroupName:
+    description: Postgres resource group name
+    required: true
+  PostgresServerName:
+    description: Flexible Postgres server name
+    required: true
+
+runs:
+  using: composite
+
+  steps:
+  - uses: Azure/login@v1
+    with:
+      creds: ${{ inputs.AzureCredentials }}
+      enable-AzPSSession: true
+
+  - name: Run script to remove extra firewall rules
+    uses: azure/powershell@v1
+    with:
+      inlineScript: |
+        $Rules = Get-AzPostgreSqlFlexibleServerFirewallRule -ResourceGroupName $Env:RESOURCE_GROUP_NAME -ServerName $Env:POSTGRES_SERVER_NAME
```

Comment:
Is it necessary to map the inputs to environment variables?  Can they be called directly as you would in a `run` step?